.github/workflows/sdk-update-api-spec.yaml

```
@@ -0,0 +1,38 @@
+name: SDK Update API Spec
+
+on:
+  schedule:
+    - cron: '0 12 * * 3'
+  workflow_dispatch: {}
+
+jobs:
+  api-spec-update:
+    runs-on: ubuntu-20.04
+    if: ${{ github.repository == 'jellyfin/jellyfin-sdk-kotlin' }}
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Setup Gradle cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-java-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-java-gradle-
+            ${{ runner.os }}-java-
+      - name: Update generated Jellyfin API spec
```

```suggestion
      - name: Update generated Jellyfin API from spec
```
Same for the others, I think this is technically more correct?